.github/workflows/ci.yml

```diff
@@ -1256,7 +1256,7 @@ jobs:
       env:
         PGP_SECRET: ${{ secrets.PGP_SECRET }}
     - run: ./mill -i ci.setShouldPublish
-    - run: ./mill -i publishSonatype '{__[],_}.publishArtifacts'
+    - run: ./mill -i publishSonatype '{__[],_,test-runner[2.13.13],test-runner[2.12.19],runner[2.13.13],runner[2.12.19]}.publishArtifacts'
```

Comment:
I wish there was a simpler way to do this

Reply:
My heart bleeds :<

Could we add a custom task that does it instead?

Reply:
perhaps.
I'll try to find the time to refactor this in a separate PR later.